.gitlab-ci.yml

```diff
@@ -1,59 +0,0 @@
-image: "julia:1"
```

Comment:
please don't remove our GPU CI. We know it's down right now (since we can't get on campus), but that isn't a reason to delete it.